cmd/root.go

```diff
@@ -150,3 +162,44 @@ func getHome() (string, error) {
 
 	return "", errors.New("Can't get home directory")
 }
+
+func generateRandByteSlice(length int) []byte {
+	b := make([]byte, length)
+	for i := range b {
+		b[i] = letterBytes[rand.Intn(len(letterBytes))]
+	}
+	return b
+}
+
+func populateSecretWithPasswords(cli kubernetes.Interface, ns, secretID string, passwordFields []string) error {
+	prevSecret, err := cli.CoreV1().Secrets(ns).Get(secretID, metav1.GetOptions{})
+	if err != nil {
+		if k8sErrors.IsNotFound(err) {
+			data := make(map[string][]byte)
+			for _, p := range passwordFields {
+				data[p] = generateRandByteSlice(10)
+			}
+			secret := v1.Secret{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      secretID,
+					Namespace: ns,
+					// TODO: Add labels to be able to clean it up
```

Comment:
To be exact, we will add the kubecfg's gc-tag annotation to this object, then the GC-engine from kubecfg can be aware of it.